guides/v2.1/config-guide/bootstrap/mage-dirs.md

```diff
@@ -33,16 +33,18 @@ You can set `MAGE_DIRS` in any of the following ways:
 	require __DIR__ . '/app/bootstrap.php';
 	$params = $_SERVER;
 	$params[Bootstrap::INIT_PARAM_FILESYSTEM_DIR_PATHS] = [
-       DirectoryList::CACHE => [DirectoryList::PATH => '/mnt/nfs/cache'],
-       DirectoryList::MEDIA => [DirectoryList::PATH => '/mnt/nfs/media', DirectoryList::URL_PATH => ''],
-	];
-
-	$params[Bootstrap::INIT_PARAM_FILESYSTEM_DIR_PATHS] = [
 	DirectoryList::PUB => [DirectoryList::URL_PATH => ''],	
 	DirectoryList::MEDIA => [DirectoryList::URL_PATH => 'media'],
 	DirectoryList::STATIC_VIEW => [DirectoryList::URL_PATH => 'static'],
 	DirectoryList::UPLOAD => [DirectoryList::URL_PATH => 'media/upload'],
 	];
+	
+	$params[Bootstrap::INIT_PARAM_FILESYSTEM_DIR_PATHS][DirectoryList::CACHE] = [
+	    DirectoryList::PATH => '/mnt/nfs/cache'
+	];
+	$params[Bootstrap::INIT_PARAM_FILESYSTEM_DIR_PATHS][DirectoryList::MEDIA] = [
+	    DirectoryList::PATH => '/mnt/nfs/media', DirectoryList::URL_PATH => ''
+	];
```

Comment:
I'd add these two keys to the array above, so it's just one place. And remove original `MEDIA` as it will be overwritten anyways. Like:
```
$params[Bootstrap::INIT_PARAM_FILESYSTEM_DIR_PATHS] = [
 	DirectoryList::PUB => ...,	
 	DirectoryList::STATIC_VIEW =>  ...,
 	DirectoryList:: MEDIA => ...,
 	DirectoryList:: CACHE => ...,
 	DirectoryList::UPLOAD => ...,
]
```

Reply:
@buskamuza understood... I'll revise this PR and push it.